### PR TITLE
Edit: Getting started

### PR DESCRIPTION
**Pull request by Mocab**
I have updated the Java version of this guide.

### CODE REVIEW

**Review by KorbsStudio — APPROVED** [review]

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 1]
```diff
@@ -16,150 +16,51 @@ authorGitHub: korbsstudio
...
 
-To make a Java version the default one, so that you don't need to select it every time you start your server, visit the [game panel](https://panel.falixnodes.net/), find and click on your server, find the "Startup" page on the top bar, scroll down to "Variables", then find "JAVA VERSION" and type the following for:
+## Setting Up The Server
 
-## Java8
-`adopt@1.8.0-275`
+1. Go to the [Game Panel](https://panel.falixnodes.net).
+2. Select your server.
+3. Click on the start button, which is located at the upper left corner of the Console tab.
+4. You will be greeted with a message asking you to choose a server type. In this case Minecraft Java. Type in the number beside the option given, for example `1`.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:26]:
```suggestion
4. You will be greeted with a message asking you to choose a server type. In this example, Minecraft Java. Type in the number beside the option you want. For example, enter `1` to select option 1.
```

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 2]
```diff
@@ -16,150 +16,51 @@ authorGitHub: korbsstudio
...
+5. You will be asked to choose a Server Software, if you don't know which to pick read our [Server Software guide](https://help.falixnodes.net/minecraft/general/server-softwares/).
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:27]:
```suggestion
5. You will be asked to choose a Server Software, if you don't know which to pick, read our [Server Software guide](https://help.falixnodes.net/minecraft/general/server-softwares/).
```

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 3]
```diff
@@ -16,150 +16,51 @@ authorGitHub: korbsstudio
...
+6. You will be prompted to choose a Minecraft version, for example `1.17.1`.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:28]:
```suggestion
6. You will be prompted to choose a Minecraft version. For example, `1.17.1`.
```

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 4]
```diff
@@ -16,150 +16,51 @@ authorGitHub: korbsstudio
...
-
-
-# Configuring
-## Message of the Day
-![Message of the Day Example](https://i.imgur.com/ctD8iqS.png)
+### Manually Setting A Java Version
 
-Also known as MOTD, is the message that shows up below the server name on a multiplayer server list. Usually used to say what's new about the server and/or also displaying what version it supports. Sometimes also used to indicate what games it has to offer or just a short description of the server.
+1. Visit the [Game Panel](https://panel.falixnodes.net).
+2. Find and click on your server, make sure it's off.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:36]:
```suggestion
2. Find and click on your server, make sure it's offline.
```

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 5]
```diff
@@ -16,150 +16,51 @@ authorGitHub: korbsstudio
...
+3. Go to the "STARTUP" tab.
+4. Under "JAVA VERSION", insert in a Java version:
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:38]:
```suggestion
4. Under "JAVA VERSION", insert a Java version:
```

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 6]
```diff
@@ -16,150 +16,51 @@ authorGitHub: korbsstudio
...
 
-To configure MOTD(Message of the Day), the setting for this is usually found in the server.properties file. Simply just change it to what you want for your Minecraft Java Server.
+    - if your server is **1.12** or below then type `adopt@1.8.0-275`.
+    - if your server is **1.13** 1.16 then type `adopt@1.11.0-9`.
+    - if your server is **1.17** then type `adopt@1.16.0-1`.
+    - if your server is **1.18** then type `1.17.0-custom=tgz+https://pterodactyl-api.catactyl.xyz/static-files/OpenJDK17U-jdk_x64_linux_hotspot_17.0.1_12.tar.gz`.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:43]: Have you tested this works and is a 'permanent' solution?

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2022-01-29-getting-started.md (hunk 6 above)
Comment (Mocab) [reply _posts/minecraft/general/2022-01-29-getting-started.md:43]: I got all the info there from exo, I will double check tomorrow morning.

Comment (Mocab) [reply _posts/minecraft/general/2022-01-29-getting-started.md:43]: I have double checked everything, it works perfectly fine.

File: _posts/minecraft/general/2022-01-29-getting-started.md [hunk 7]
```diff
@@ -16,150 +16,52 @@ authorGitHub: korbsstudio
...
 
-If you're interested in changing text colors, use these color codes:
+5. Go back to the console and start the server.
+> If this does not work, manually delete the `.jabba` folder in the File manager then start your server.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2022-01-29-getting-started.md:46]:
```suggestion
> If this does not work, manually delete the `.jabba` folder in the file manager, then start your server.
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

**Review by DaddyChimken — APPROVED** [review]

**Pull request merged**